phpcs.xml.dist

```diff
@@ -80,17 +80,24 @@
 	<!-- Load WordPress Coding standards -->
 	<rule ref="WordPress">
 		<!-- Definitely should not be added back -->
-		<exclude name="Generic.Arrays.DisallowShortArraySyntax"/>
-		<exclude name="WordPress.CodeAnalysis.AssignmentInCondition.Found"/>
+		<exclude name="Generic.CodeAnalysis.AssignmentInCondition.Found"/>
+		<exclude name="Universal.Arrays.DisallowShortArraySyntax"/>
+		<exclude name="Universal.Operators.DisallowShortTernary.Found"/>
 		<exclude name="WordPress.Files.FileName"/>
 		<exclude name="WordPress.NamingConventions.ValidVariableName"/>
-		<exclude name="WordPress.PHP.DisallowShortTernary.Found"/>
 
 		<!-- This would be a breaking change to fix-->
 		<exclude name="WordPress.NamingConventions.ValidFunctionName.MethodNameInvalid" />
 
 		<!-- Should probably not be added back -->
 		<exclude name="PHPCompatibility.Keywords.ForbiddenNamesAsDeclared.objectFound"/>
+		<exclude name="WordPress.WP.Capabilities.Undetermined" />
```

Comment:
@jasonbahl your understanding is 🎯. The way to remediate this in our code would be to `phpcs:ignore` on each case. When dealing with custom capabilities that active form of confirmation is valuable, but since we're only ever referencing upstream capabilities, it seems unnecessary.

I'll relocate it to the other comment shortly.

Reply:
Done in https://github.com/wp-graphql/wp-graphql/pull/2924/commits/6a6296af165cd0fdfee205bce85851ac4c6779a3